src/RadForce/test_radiation_force.cpp

```diff
@@ -192,10 +193,11 @@ AMRSimulation<TubeProblem>::setCustomBoundaryConditions(
     consVar(i, j, k, RadSystem<TubeProblem>::x3RadFlux_index) = 0;
 
     consVar(i, j, k, RadSystem<TubeProblem>::gasDensity_index) = rho;
+    consVar(i, j, k, RadSystem<TubeProblem>::gasEnergy_index) = 0;
+    consVar(i, j, k, RadSystem<TubeProblem>::gasInternalEnergy_index) = -((rho * vel)*(rho * vel))/(2*rho);
```

Comment:
just set it to zero here. the gas internal energy can't be negative (physically).